samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/AppManifest/manifest.json

```diff
@@ -0,0 +1,48 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.13/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.13",
+  "version": "1.0.0",
+  "id": "f844cd97-926e-48b7-86fb-8cc3c6e70673",
+  "packageName": "com.teams.meetingtranscriptrecording",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Meeting Transcripts Recordings",
+    "full": "Meeting Transcripts Recordings"
+  },
+  "description": {
+    "short": "Sample Teams Meeting Transcripts and Recordings.",
+    "full": "Sample Teams Meeting Transcripts and Recordings."
+  },
+  "accentColor": "#5558AF",
+  "staticTabs": [
+    {
+      "entityId": "meetingtranscriptrecording",
+      "name": "Meeting T&R Graph API Demo",
+      "contentUrl": "https://ad43-2401-4900-606c-8175-6dac-969c-1977-75ab.ngrok-free.app/meetingtranscriptrecording",
```

Comment:
Let's put placeholders here instead of actual URL.

Reply:
Fixed